redbot/cogs/downloader/downloader.py

```diff
@@ -1405,14 +1362,50 @@ async def findcog(self, ctx: commands.Context, command_name: str) -> None:
             cog_name = self.cog_name_from_instance(cog)
             installed, cog_installable = await self.is_installed(cog_name)
             if installed:
-                msg = self.format_findcog_info(command_name, cog_installable)
-            else:
-                # Assume it's in a base cog
-                msg = self.format_findcog_info(command_name, cog)
+                from_git = True
+                made_by = ", ".join(cog_installable.author) or _("Missing from info.json")
+                repo_url = (
+                    _("Missing from installed repos")
+                    if cog_installable.repo is None
+                    else cog_installable.repo.clean_url
+                )
+                cog_name = cog_installable.name
+            elif cog.__module__.startswith("redbot."):  # core commands or core cog
+                from_git = False
+                made_by = "Cog Creators"
```

Comment:
```suggestion
                made_by = _("Cog Creators")
```

Reply:
Should that really be translated? 🤔 
Wait for Preda to make us into "Cog Créateurs" 😄

Reply:
> Should that really be translated? 🤔
> Wait for Preda to make us into "Cog Créateurs" 😄

Yeha no you are right but it also should be `Cog-Creators` and not `Cog Creators`

Reply:
> Yeha no you are right but it also should be `Cog-Creators` and not `Cog Creators`

I think it's only `Cog-Creators` in the organisation name on GitHub and everywhere else, we refer to ourselves as `Cog Creators`, examples:
`[p]info` command:
![image](https://user-images.githubusercontent.com/6032823/85034308-3f0e7b80-b182-11ea-8c9b-f17a49c3606f.png)
Copyright at the bottom of docs (https://docs.discord.red/en/latest/):
![image](https://user-images.githubusercontent.com/6032823/85034356-4d5c9780-b182-11ea-8a86-2e639b412cfe.png)

Reply:
> > Yeha no you are right but it also should be `Cog-Creators` and not `Cog Creators`
> 
> I think it's only `Cog-Creators` in the organisation name on GitHub and everywhere else, we refer to ourselves as `Cog Creators`, examples:
> `[p]info` command:
> ![image](https://user-images.githubusercontent.com/6032823/85034308-3f0e7b80-b182-11ea-8c9b-f17a49c3606f.png)
> Copyright at the bottom of docs (https://docs.discord.red/en/latest/):
> ![image](https://user-images.githubusercontent.com/6032823/85034356-4d5c9780-b182-11ea-8a86-2e639b412cfe.png)

Blame Kowlin for saying `Cog-Creators`